gruntfileConfig.json

```diff
@@ -4,6 +4,7 @@
   "src": "./src",
   "assets": "./src/assets",
   "theme": "./node_modules/patternpack-example-theme",
+  "logo": "/theme-assets/images/pp-logo.svg",
```

Comment:
You are correct, however you are thinking from the perspective of someone using the theme.

If you are authoring the theme, because you hardcoded `"logo": "/theme-assets/images/pp-logo.svg",` into the configuration.  All themes will by default use `pp-logo.svg` as their default logo name.  If I don't want each pattern library to have to configure a special name, I (as a theme author) must use `pp-logo.svg` as my logo name, so that the patternpack defaults will work.